docs/user_guide/ui_customization.md

```diff
@@ -0,0 +1,170 @@
+# UI Customization
+
+This guide explains how to customize the user interface of the geOrchestra Gateway, including login pages, error pages, and static resources.
+
+## Main Principles
+
+To customize templates like the login page or error pages:
+
+1. Copy the template files from `gateway/src/main/resources/templates` to a custom directory
+2. Modify the templates to suit your needs
+3. Configure the Gateway to use the custom templates
+
+```yaml
+spring:
+  thymeleaf:
+    prefix: file:${georchestra.datadir}/gateway/templates/
+```
+
+Create a `login.html` file in this directory to customize login, and a `logout.html` file for logout.
+
+## Header Customization
+
+The geOrchestra header is integrated by default on the Gateway login page using the `<geor-header></geor-header>` HTML tag.
+
+The web component is loaded from a JavaScript file defined in the `default.properties` file:
+
+```properties
+# From georchestra datadir's default.properties
+# URL to the Javascript definition of the <geor-header> web component
+headerScript: https://cdn.jsdelivr.net/gh/georchestra/header@dist/header.js
+
+# From the gateway's yaml configuration
+# includes or disables the <geor-header/> web component
+georchestra.gateway.headerEnabled: true
+```
+
+You can customize the header by:
+
+1. Creating your own version of the header component
+2. Updating the `headerScript` property to point to your custom implementation
+3. Or completely disabling the header by setting `georchestra.gateway.headerEnabled: false`
+
```

Comment:
Maybe mention that you can now [configure the header's menu entries](https://github.com/georchestra/header/blob/main/CONFIG.md)

Reply:
Would that be part of the gateway configuration or the header application?

Reply:
That's configuring the header, in the gateway application. It's mostly heaer-related, but worth mentioning here. Probably just mentioning it with a href to the header doc would be enough, I'd say